sys/color/color.c

```diff
@@ -127,3 +127,15 @@ void color_hsv2rgb(color_hsv_t *hsv, color_rgb_t *rgb)
             break;
     }
 }
+
+void color_hex2rgb(uint32_t hex, color_rgb_t *rgb)
+{
+    rgb->r = ((hex >> 16) & 0xFF);
+    rgb->g = ((hex >> 8) & 0xFF);
+    rgb->b = (hex & 0xFF);
```

Comment:
Hacky, but more afficienent way could be

```C
uint32_t *ptr = (uint32_t *)rgb;
*ptr = hex << 8;
```

(please test this if you decide to use this. Might be, that I'm wrong about the shift.